src/libs/OptionsListUtils.js

```diff
@@ -96,7 +99,21 @@ function createOption(personalDetailList, report, draftComments, {showChatPrevie
         + _.unescape(report.lastMessageText)
         : '';
     const tooltipText = getReportParticipantsTitle(lodashGet(report, ['participants'], []));
-
+    const participantNames = new Set();
```

Comment:
In [this commit](https://github.com/Expensify/Expensify.cash/pull/2958/commits/ec3f9d6c5646a96ca0368f717fcbaf037fe40905) you changed the names from `participantSet`	to `participantNames` but you're still creating a `new Set()` and returning it, which is, I think, what @marcaaron took issue with.

Reply:
It's still intended to be a Set, since Sets allow constant time lookup, but I'll change it to an object and use it like a dictionary.